docs/src/locators.md

```diff
@@ -10,13 +10,13 @@ a way to find element(s) on the page at any moment.
 
```

Comment:
Mass comments to the snippets:
- Remove line breaks in python snippets;
- C# string options are called `NameString/NameRegex` and `HasTextString/HasTextRegex`.